### PR TITLE
Updated tokenUrl and uploadUrl config props for CKEditor Cloud Services

### PR DESCRIPTION
**Pull request by mlewand**
A followup to ckeditor/ckeditor-dev#1760.

### CODE REVIEW

File: docs/guide/dev/integration/cloudservices/README.md [hunk 1]
```diff
@@ -22,8 +22,8 @@ The plugin exports {@linkapi CKEDITOR.plugins.cloudservices.cloudServicesLoader}
 
 CKEditor Cloud Services requires two configuration options to be set:
 
-* {@linkapi CKEDITOR.config.cloudServices_url} &ndash; The URL to the cloud services upload endpoint. Please refer to the [official CKEditor Cloud Services documentation about endpoints](https://docs.ckeditor.com/cs/latest/guides/token-endpoints/tokenendpoint.html) for more information.
-* {@linkapi CKEDITOR.config.cloudServices_token} &ndash; The authorization token needed to use the upload endpoint, unique to your CKEditor Cloud Services account.
+* {@linkapi CKEDITOR.config.cloudServices_uploadUrl} &ndash; The URL to the cloud services upload endpoint. Please refer to the [official CKEditor Cloud Services documentation about endpoints](https://docs.ckeditor.com/cs/latest/guides/token-endpoints/tokenendpoint.html) for more information.
```
Comment (Comandeer) [inline docs/guide/dev/integration/cloudservices/README.md]: Maybe "Cloud Services", not "cloud services"?

File: docs/guide/dev/integration/cloudservices/README.md [hunk 2]
```diff
@@ -22,8 +22,8 @@ The plugin exports {@linkapi CKEDITOR.plugins.cloudservices.cloudServicesLoader}
...
+* {@linkapi CKEDITOR.config.cloudServices_tokenUrl} &ndash; The authorization token needed to use the upload endpoint, unique to your CKEditor Cloud Services account.
```
Comment (Comandeer) [inline docs/guide/dev/integration/cloudservices/README.md]: It's not a token anymore, but a URL.

**Comment by Comandeer** [comment]
I've pushed proposed changes.

**Comment by mlewand** [comment]
I just removed this one extra sentence in 0def362 as it's redundant (duplicating token URL docs). Other than that all good.

**Pull request merged**